Add new option `parse_animation` to parse metadata for animated images (#676)
FlutterGen/flutter_gen: README.md

}
```

> When we need to obtain more animation details, we can use the `parse_animation` option. This will automatically parse all animation information for GIFs and WebP files, including frames, duration, etc. As this option significantly increases generation time, it's disabled by default and needs to be manually enabled.

Reviewer: ```suggestion
You can use `parse_animation` to generate more animation details.
It will automatically parse all animation information for GIF and WebP files,
including frames, duration, etc. As this option significantly increases generation time,
The option is disabled by default; enabling it will significantly increase the generation elapse.
```

```yaml
flutter_gen:

>   parse_animation: true # <- Add this line (default: false)
>                         # This option implies parse_metadata: true

Reviewer: ```suggestion
  parse_animation: true # <- Add this line (default: false)
  # This option implies parse_metadata: true
```

synthetic-package: false <--- ⚠️Add this line⚠️
```

> If you get

Reviewer: It looks like a mistake in the original doc.
```suggestion
```
